fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -2102,18 +2105,18 @@ public static QueryParameter parseChainedInclusionCriteria(QueryParameter inclus
 
     /**
      * Normalizes a string to be used as a search parameter value. All accents and
-     * diacritics are removed. And then the
-     * string is transformed to lower case.
+     * diacritics are removed. Consecutive whitespace characters are replaced with
+     * a single space. And then the string is transformed to lower case.
      *
-     * @param value
-     * @return
+     * @param value the string to normalize
+     * @return the normalized string
      */
     public static String normalizeForSearch(String value) {
 
         String normalizedValue = null;
         if (value != null) {
             normalizedValue = Normalizer.normalize(value, Form.NFD).replaceAll("\\p{InCombiningDiacriticalMarks}+", "");
-            normalizedValue = normalizedValue.toLowerCase();
+            normalizedValue = normalizedValue.replaceAll("\\s+", " ").toLowerCase();
```

Comment:
Curious why the removal of consecutive whitespace characters - is that standard? I assume this affects string search as well as token :text search.

Reply:
In the FHIR spec, https://www.hl7.org/fhir/search.html#string, it mentions that "Punctuation and non-significant whitespace (e.g. repeated space characters, tab vs space) should also be ignored."  This change was to normalize all whitespace to be handled the same.